living_documentation_generator/action_inputs.py

```diff
@@ -15,19 +15,16 @@
 #
 
 """
-This module contains the ActionInputs class which is responsible for loading, managing and validating the inputs
-required for running the GH Action.
+This module contains an Action Inputs class methods,
+which are essential for running the GH action.
 """
 
 import json
 import logging
 import sys
 
 from living_documentation_generator.model.config_repository import ConfigRepository
-from living_documentation_generator.utils.utils import (
-    get_action_input,
-    make_absolute_path,
-)
+from living_documentation_generator.utils.utils import get_action_input, make_absolute_path
```

Comment:
From how many import items are required ()?

Reply:
Whenever line is more than 120 char. The rules are still same. BUT If we talking about for example constants, which can increase during the time, I also think is alright to put them under each other. Black way of formatting is to reduce as many extra lines as possible. But it won't change your code, if you choose to put just two of your imports under each other.